lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -409,8 +411,15 @@ export abstract class CacheManager implements ICacheManager {
                     return;
                 }
 
+                // KeyId (sshKid) in request must match cached SSH certificate keyId
+                if(tokenType === AuthenticationScheme.SSH) {
+                    if(!!keyId && !this.matchKeyId(entity, keyId)) {
```

Comment:
Dont think this is necessary
```suggestion
                    if(keyId && !this.matchKeyId(entity, keyId)) {
```